ui/appui-react/src/appui-react/widgets/WidgetUtilities.tsx

```diff
@@ -0,0 +1,38 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Widget
+ */
+
+import type { StagePanelLocation } from "../stagepanels/StagePanelLocation";
+import type { StagePanelSection } from "../stagepanels/StagePanelSection";
+
+/** Helper class to create widget items.
+ * @alpha
+ */
+export namespace WidgetUtilities {
+  /** @alpha */
+  export function toContainerId( // TODO: rename?
```

Comment:
Same reasoning as `ToolbarItemUtilities` OR might need a rename either way due to `containerId` rename.